src/test/java/no/unit/nva/handlers/AddUserTest.java

```diff
@@ -133,22 +129,6 @@ public void processInputThrowsConflictExceptionWhenInputUserDoesNotHaveUsername(
         assertThat(response.getStatusCode(), is(equalTo(HttpStatus.SC_BAD_REQUEST)));
     }
 
-    @DisplayName("processInput() throws RuntimeException when database service throws unexpected exception "
-        + "when getting user ")
-    @Test
-    public void processInputThrowsUnexpectedExceptionWhenDatabaseServiceThrowsUnexpectedExceptionWhenReadingUser()
```

Comment:
This test was testing the case where the valid data sent to the database are read again and now they are invalid. This basically tests the rare case where there is an error "on the wire" when getting the data.   There is little point on testing this extreme case. The only reason this test was here was for test coverage.